.github/workflows/deploy.py

```diff
@@ -0,0 +1,220 @@
+    else:
+        click.echo("The image is not published, please create tag for publishing")
+
+
+def run_subprocess(command):
+    click.echo(f"run command: {command}")
+    subprocess.run(command, shell=True)
+
+
+@cli.command(name="run_tests")
+@click.option('--github_sha')
+def run_tests(github_sha):
+    image_name = f"{IMAGE_NAME}:{github_sha}"
+    os.environ["EVM_LOADER_IMAGE"] = image_name
+    run_subprocess(f"docker-compose -f ./evm_loader/docker-compose-test.yml down")
+    run_subprocess(f"docker-compose -f ./evm_loader/docker-compose-test.yml up -d")
+
+    try:
+        click.echo("Start tests")
+        exec_id = docker_client.exec_create(
+            container="solana", cmd="/opt/deploy-test.sh")
+        logs = docker_client.exec_start(exec_id['Id'])
+        click.echo(f'logs: {logs}')
+        for line in logs:
+            if 'ERROR ' in str(line) or 'FAILED ' in str(line):
+                raise RuntimeError("Test are failed")
+    except:
+        raise RuntimeError("Solana container is not run")
+
+
+@cli.command(name="stop_containers")
+def stop_containers():
+    run_subprocess(f"docker-compose -f ./evm_loader/docker-compose-test.yml down")
+
+
+@cli.command(name="trigger_proxy_action")
+@click.option('--head_ref_branch')
+@click.option('--base_ref_branch')
+@click.option('--github_ref')
+@click.option('--github_sha')
+@click.option('--token')
+@click.option('--is_draft')
+@click.option('--labels')
+def trigger_proxy_action(head_ref_branch, base_ref_branch, github_ref, github_sha, token, is_draft, labels):
+    is_develop_branch = github_ref in ['refs/heads/develop', 'refs/heads/master']
+    is_tag_creating = 'refs/tags/' in github_ref
+    is_version_branch = re.match(VERSION_BRANCH_TEMPLATE, github_ref.replace("refs/heads/", "")) is not None
+    is_FTS_labeled_not_draft = 'FullTestSuit' in labels and is_draft != "true"
+
+    if is_develop_branch or is_tag_creating or is_version_branch or is_FTS_labeled_not_draft:
+        full_test_suite = "true"
+    else:
+        full_test_suite = "false"
+
+    github = GithubClient(token)
+
+    if head_ref_branch in github.get_proxy_branches():
+        proxy_branch = head_ref_branch
+    elif re.match(VERSION_BRANCH_TEMPLATE, base_ref_branch):
+        proxy_branch = base_ref_branch
+    elif is_tag_creating:
+        proxy_branch = github_ref.replace("refs/tags/", "")
+    elif is_version_branch:
+        proxy_branch = github_ref.replace("refs/heads/", "")
+    else:
+        proxy_branch = 'develop'
+    click.echo(f"Proxy branch: {proxy_branch}")
+
+    runs_before = github.get_proxy_runs_list(proxy_branch)
+
+    github.run_proxy_dispatches(proxy_branch, github_ref, github_sha, full_test_suite)
+    wait_condition(lambda: len(github.get_proxy_runs_list(proxy_branch)) > len(runs_before))
+
+    runs_after = github.get_proxy_runs_list(proxy_branch)
+    proxy_run_id = list(set(runs_after) - set(runs_before))[0]
+    click.echo(f"Proxy run id: {proxy_run_id}")
```

Comment:
Please print a link to the proxy test run before waiting. It is helpful for debugging.